rs.geo_map.view.lkml

```diff
@@ -100,15 +106,20 @@ view: rs_logrecno_bg_map {
   dimension: county {
     group_label: "County"
     label: "County FIPS Code"
-    sql: CONCAT(${state}, ${TABLE}.county_fips_code);;
+    sql: (${state} || ${TABLE}.county_fips_code)
```

Comment:
I was working on converting/using this data set for our own team internally and was wondering why this concat was even happening in the first place. Can anyone enlighten me?

Reply:
@allegramholland can definitely offer more insight to the "why,"  but I edited the syntax of the concatenation to match the correct redshift syntax.